lib/board.rb

```diff
@@ -1,2 +1,62 @@
 class Board
+  #1) how to populate gird with ships
+  #2) need to display board and update
+
+  attr_accessor :grid
+
+  def initialize(grid = Board.default_grid)
+    @grid = grid
+  end
+
+  def self.default_grid
+    Array.new(10) { Array.new(10)}
+  end
+
+  def [](pos)
+    row, col = pos
+    grid[row][col]
+  end
+
+  def over?
+    grid.flatten.none? { |pos| pos.nil? }
+  end
+
+  def populate_grid
+#how many ships do we populate with? 5
+#create random size
+#how to randomly place on grid, mark with :s
+#need help
+    #rand_ship_size = rand(1..grid.length)
+    #rand_ship_start = [rand(1..grid.length), rand(1..grid.length)
+    #5.times do |ship|
+      #ship = grid[rand_ship_start] == :s if grid[rand_start.empty?]
+    #end
```

Comment:
Ok here we need to places N ships on the grid, you have
- the first element of an array has the index 0: 
```ruby
arr = [:a, :b]
arr[0]
# :a
```

- you need to generate a new position for each ship